addons/sale/report/sale_report_views.xml

```diff
@@ -122,7 +122,7 @@
     <record id="action_order_report_all" model="ir.actions.act_window">
         <field name="name">Sales Analysis</field>
         <field name="res_model">sale.report</field>
-        <field name="view_mode">graph,pivot,tree</field>
+        <field name="view_mode">graph,pivot,tree,form</field>
```

Comment:
Open in Form view of report model feels pretty useless IMHO
Need to check with PO why he needs this :wink: